next-docs/pages/world/namespaces-access-control.mdx

```diff
@@ -1 +1,74 @@
 # Namespaces and Access Control
+
+MUD access control is based on namespaces.
+Every resource is part of a namespace (if there is no namespace specified that just mean that the resource is part of the root namespace).
+Access to every resource is managed by the owner address of the namespace.
+
+## Resource identifiers
+
+A `ResourceId` is a 32-byte value that uniquely identifies a resource in a MUD `World`.
+It is two bytes of resource type followed by 14 bytes of namespace and then 16 bytes of the name of the actual resource.
+
+Currently, MUD supports these resource types:
+
+- [Table (`tb`)](https://github.com/latticexyz/mud/blob/main/packages/store/src/storeResourceTypes.sol#L11).
+  An onchain table whose information that is available both onchain (through calls to `view` functions) and offchain (either through calls to `view` functions, events, or an [indexer](/services/indexer)).
+- [Offchain table (`ot`)](https://github.com/latticexyz/mud/blob/main/packages/store/src/storeResourceTypes.sol#L14).
+  An offchain table whose information is only available offchain (either through events, or through an [indexer](/services/indexer)).
+- [Namespace (`ns`)](https://github.com/latticexyz/mud/blob/main/packages/world/src/worldResourceTypes.sol#L15).
+  A namespace is a container for tables, offchain tables, and systems, as explained here.
+- [System (`sy`)](https://github.com/latticexyz/mud/blob/main/packages/world/src/worldResourceTypes.sol#L25).
+  A system contains functions that interact with table data onchain.
+- [Module (`md`)](https://github.com/latticexyz/mud/blob/main/packages/world/src/worldResourceTypes.sol#L21).
+  A module is a script that can install tables, systems, etc.
+
+Access control can be attached to any resource.
+
+## Access control levels
+
+There are three access levels in MUD.
+
+### "No" access
+
+This is the level that most users of a MUD application have for all resources.
+Users on this level are allowed to do the following:
+
+- Read information (table content).
+  _There are no secrets on the blockchain_, transactions and events are public information, so this information has to be available.
```

Comment:
> There are no secrets on the blockchain

This point is arguable (zk, hash commitments, etc), would remove it here

```suggestion
```

Reply:
The secret information isn't *on the blockchain*. But I'll make it clearer.